app/controllers/api/v1/dependencies_controller.rb

```diff
@@ -0,0 +1,34 @@
+class Api::V1::DependenciesController < Api::BaseController
+  skip_before_action :verify_authenticity_token
+  GEM_REQUEST_LIMIT = 250
+
+  def index
+    return render text: '' if gem_names.empty?
```

Comment:
Yes this needs to return json if the format is json. https://github.com/bundler/bundler-api/blob/8af966db029364911288a0e03224b92ecbeec298/lib/bundler_api/web.rb#L119-L122